app/src/main/res/values/strings.xml

```diff
@@ -248,6 +248,7 @@
     <string name="clipboard" tools:keep="@string/clipboard">Clipboard</string>
     <string name="clear_clipboard" tools:keep="@string/clear_clipboard">Clear clipboard</string>
     <string name="voice" tools:keep="@string/voice">Voice input</string>
+    <string name="numpad" tools:keep="@string/numpad">@string/layout_numpad</string>
```

Comment:
Could you add `translatable="false"` here? Translations are already done on `layout_numpad`.